libraries/Microsoft.Bot.Schema/Teams/Surface.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Bot.Schema.Converters;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace Microsoft.Bot.Schema.Teams
+{
+    /// <summary>
+    /// Specifies where the signal will be rendered in the meeting UX.
+    /// </summary>
+    [JsonConverter(typeof(SurfaceConverter))]
+    public class Surface
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Surface"/> class.
+        /// </summary>
+        /// <param name="type">Type of Surface.</param>
+        public Surface(SurfaceType type)
+        {
+            Type = type;
+        }
+
+        /// <summary>
+        /// Gets or sets Surface type, the value indicating where the signal will be rendered in the meeting UX.
```

Comment:
nit: instead of signal, let's use notification.

Reply:
Will update in the next iteration